tests/cluster/schema_manager/test_schema_manager.py

```diff
@@ -5,14 +5,20 @@
 from atc.configurator import Configurator
 from atc.delta import DeltaHandle
 from atc.schema_manager import SchemaManager
-from tests.cluster.delta.SparkExecutor import SparkSqlExecutor
```

Comment:
@Okami1 I used relative imports here, to highlight the fact that I am importing stuff from within the current test folder. This also allowed me to discover the mistake that you imported the SqlExecutor from the delta tests instead of your own.